storage-reverse-image-search/README.md

```diff
@@ -88,7 +88,7 @@ This extension uses other Firebase and Google Cloud Platform services, which hav
 
 * Approximate number of neighbors: The approximate number of neighbors to return in the response. [Read more about this parameter here](https://cloud.google.com/vertex-ai/docs/matching-engine/configuring-indexes#nearest-neighbor-search-config).
 
-* Batch size: The batch size used to generate feature vectors for existing images. The larger the batch size, the more time and memory are required. Do not sit a size larger than 64.
+* Batch size: The batch size used to generate feature vectors for existing images. The larger the batch size, the more time and memory are required. Do not set a size larger than 64.
```

Comment:
Apply this fix to [PREINSTALL.md](https://github.com/GoogleCloudPlatform/firebase-extensions/blob/next/storage-reverse-image-search/PREINSTALL.md)

Reply:
I could not find the `npm run generate-readme` command for some reason

Reply:
I did push the update to s/sit/set

Reply:
LGTM thanks!

You can find it in [functions/package.json](https://github.com/GoogleCloudPlatform/firebase-extensions/blob/ea2ed5c2d4246be309a33e26428d16fdd6de18c7/palm-secure-backend/functions/package.json#LL9C6-L9C21)